django/utils/timesince.py

```diff
@@ -26,9 +35,15 @@ def timesince(d, now=None, reversed=False):
     displayed.  For example, "2 weeks, 3 days" and "1 year, 3 months" are
     possible outputs, but "2 weeks, 3 hours" and "1 year, 5 days" are not.
 
+    `time_strings` is an optional dict of strings to replace the default
```

Comment:
I don't see a use case for the new argument right now. If you want to replace the strings, it seems more commen to provide your own PO gettext files/translations. That way you can provide your own strings, while not changing the place holders.

Reply:
aaah, ok, now I get it.